app/models/filter.rb

```diff
@@ -32,14 +32,19 @@ def cards
       result = result.where(cards: { created_at: creation_window }) if creation_window
       result = result.closed_at_window(closure_window) if closure_window
       result = result.closed_by(closers) if closers.present?
-      result = terms.reduce(result) do |result, term|
-        result.mentioning(term, user: creator)
-      end
-
+      result = result.mentioning(combined_fts_query, user: creator) if terms.present?
       result.distinct
     end
   end
 
+  def combined_fts_query
+    terms.map { |term| sanitize_fts_term(term) }.join(' AND ')
```

Comment:
will the `AND` and the sanitization logic work for MySQL FTS too?

cc @djmb in case he has insight here.